File: android/app/src/main/java/com/dam/starwarsapp/domain/model/Film.kt
package com.dam.starwarsapp.domain.model

data class Film(
    val id: Int,
    val title: String,
    val episodeId: Int,
    val openingCrawl: String,
    val director: String,
    val producer: String,
    val releaseDate: String,
    val characterCount: Int,
    val planetCount: Int,
    val starshipCount: Int,
    val vehicleCount: Int,
    val speciesCount: Int,
    val created: String,
    val edited: String,
    val url: String,
)

File: iOS/Star-Wars-Demo-APP/Star-Wars-Demo-APP/Domain/Model/Film.swift
import Foundation

struct Film: Identifiable, Equatable, Hashable {
    let id: Int
    let title: String
    let episodeId: Int
    let openingCrawl: String
    let director: String
    let producer: String
    let releaseDate: String
    let charactersCount: Int
    let planetsCount: Int
    let starshipsCount: Int
    let vehiclesCount: Int
    let speciesCount: Int
}
